pkg/antctl/raw/multicluster/deploy/deploy_helper.go

```diff
@@ -0,0 +1,186 @@
+	}
+
+	return manifests, nil
+}
+
+func createResources(cmd *cobra.Command, content []byte) error {
+	kubeconfig, err := raw.ResolveKubeconfig(cmd)
+	if err != nil {
+		return err
+	}
+	restconfigTmpl := rest.CopyConfig(kubeconfig)
+	raw.SetupKubeconfig(restconfigTmpl)
+
+	k8sClient, err := kubernetes.NewForConfig(kubeconfig)
+	if err != nil {
+		return err
+	}
+	dynamicClient, err := dynamic.NewForConfig(kubeconfig)
+	if err != nil {
+		return err
+	}
+
+	decoder := yamlutil.NewYAMLOrJSONDecoder(bytes.NewReader([]byte(content)), 100)
+	for {
+		var rawObj runtime.RawExtension
+		if err = decoder.Decode(&rawObj); err != nil {
+			break
+		}
+
+		obj, gvk, err := yaml.NewDecodingSerializer(unstructured.UnstructuredJSONScheme).Decode(rawObj.Raw, nil, nil)
+		if err != nil {
+			return err
+		}
+		unstructuredMap, err := runtime.DefaultUnstructuredConverter.ToUnstructured(obj)
+		if err != nil {
+			return err
+		}
+
+		unstructuredObj := &unstructured.Unstructured{Object: unstructuredMap}
+
+		gr, err := restmapper.GetAPIGroupResources(k8sClient.Discovery())
+		if err != nil {
+			return err
+		}
+
+		mapper := restmapper.NewDiscoveryRESTMapper(gr)
+		mapping, err := mapper.RESTMapping(gvk.GroupKind(), gvk.Version)
+		if err != nil {
+			return err
+		}
+
+		var dri dynamic.ResourceInterface
+		if mapping.Scope.Name() == meta.RESTScopeNameNamespace {
+			dri = dynamicClient.Resource(mapping.Resource).Namespace(unstructuredObj.GetNamespace())
+		} else {
+			dri = dynamicClient.Resource(mapping.Resource)
+		}
+
+		if _, err := dri.Create(context.TODO(), unstructuredObj, metav1.CreateOptions{}); err != nil {
+			if !kerrors.IsAlreadyExists(err) {
+				return err
+			}
+		}
+		fmt.Fprintln(cmd.OutOrStdout(), fmt.Sprintf("%s/%s created", unstructuredObj.GetKind(), unstructuredObj.GetName()))
+	}
+
+	return nil
+}
+
+func deploy(cmd *cobra.Command, role string, version string, namespace string, filename string) error {
+	if filename != "" {
+		content, err := ioutil.ReadFile(filename)
+		if err != nil {
+			return err
+		}
+		if err := createResources(cmd, content); err != nil {
+			return err
+		}
+	} else {
+		manifests, err := generateManifests(role, version)
+		if err != nil {
+			return err
+		}
+		for _, manifest := range manifests {
+			// #nosec G107
+			resp, err := http.Get(manifest)
+			if err != nil {
+				return err
+			}
+			b, err := ioutil.ReadAll(resp.Body)
+			if err != nil {
+				return err
+			}
+
+			content := string(b)
+			if role == leaderRole && strings.Contains(manifest, "namespaced") {
+				content = strings.ReplaceAll(content, "changeme", namespace)
+			} else if role == memberRole && strings.Contains(manifest, "member") {
+				content = strings.ReplaceAll(content, "kube-system", namespace)
+			}
+
+			if err := createResources(cmd, []byte(content)); err != nil {
+				return err
+			}
+		}
+	}
+	fmt.Fprintln(cmd.OutOrStdout(), fmt.Sprintf("The %s cluster resources are deployed", role))
```

Comment:
the role is `leaderCluster` or `memberCluster`, which means this output will be like `The leaderCluster cluster resources are deployed`, maybe just name the role as `leader` and `member`?